package.json

```diff
@@ -20,7 +20,7 @@
     "publish": "lerna publish --conventional-commits",
     "reinstall": "yarn clean && yarn install",
     "start": "yarn watch",
-    "test": "yarn build && cross-env HOME=$PWD LANG=en_US.UTF-8 NO_COLOR=1 vitest run --coverage",
+    "test": "yarn build && cross-env HOME=$PWD LANG=en_US.UTF-8 vitest run --coverage",
```

Comment:
@marcalexiei

removing `NO_COLOR=1` is a bit unfriendly to me personally, I always have different snapshot if the env disabled.

Reply:
I removed `NO_COLOR=1` to have debug log more readable. I can restore it.